analyzer/src/test/kotlin/PackageJsonUtilTest.kt

```diff
@@ -53,71 +53,79 @@ class PackageJsonUtilTest : StringSpec() {
     }
 
     init {
-        "given no NPM lockfile present, hasNpmLockFile returns false" {
-            setupProject(path = "a")
+        "hasNpmLockFile" should {
```

Comment:
I have to say I like this less and would prefer to keep the previous because
(1) it adds even another level of identation
(2) The name of the test becomes spread to two different lines not next to each other
so one has to look further up or even scroll just for reading the full test name
(3) the given when then naming imo seems a bit more exact about what the test case actually tests

Reply:
I actually had problems understanding the purpose of a test from the previous names, which were very long, repetitive, and "reversely" worded. That's why I changed it, although I didn't initially plan to do it. I don't think the split into two lines actually is a problem, as the first line only contains a common prefix that can mostly be ignored for understand the test's purpose.